pkg/trace/api/endpoints.go

```diff
@@ -80,6 +80,10 @@ var endpoints = []endpoint{
 		Pattern: "/v0.5/traces",
 		Handler: func(r *HTTPReceiver) http.Handler { return r.handleWithVersion(v05, r.handleTraces) },
 	},
+	{
+		Pattern: "/v0.6/traces",
```

Comment:
Can we please add full documentation to the `v06` variable in `pkg/trace/api/version.go`, describing this new protocol? You can look at the documentation of `v05` for inspiration.

Reply:
What do you think of the following simple comment?

```
	// v06
	//
	// Content-Type: application/msgpack
	// Payload: TracerPayload.
	// Response: Service sampling rates.
	// 
	v06 Version = "v0.6"
```

For `v07` probably we need to explain how things work. But, in `v06` case, it just receivers `TracerPayload` in `msgpack` and, `TracerPayload` is fully documented. I think we don't need to re-explain `TracerPayload`, here.